local_planner/launch/rs_depthcloud.launch

```diff
@@ -45,14 +45,14 @@ Processing enabled by this node:
   <!-- Camera device specific arguments -->
 
   <arg name="serial_no"           default=""/>
-  <arg name="json_file_path"      default=""/>
+  <arg name="json_file_path"      default="/usr/auterion_ws/src/avoidance/local_planner/resource/stereo_calib.json"/>
```

Comment:
not sure if this belongs in the open source project

Reply:
you are right it should be relative path to the directory inside the local_planner package. Some how it did not work with relative paths at first try. But I will try to fix it